.gitlab/pipeline/zombienet/substrate.yml

```diff
@@ -50,23 +57,29 @@ zombienet-substrate-0000-block-building:
 
 zombienet-substrate-0001-basic-warp-sync:
   extends:
-    - .zombienet-substrate-common
+    - .zombienet-substrate-warp-sync-common
   script:
     - /home/nonroot/zombie-net/scripts/ci/run-test-local-env-manager.sh
       --local-dir="${LOCAL_DIR}/0001-basic-warp-sync"
       --test="test-warp-sync.zndsl"
 
 zombienet-substrate-0002-validators-warp-sync:
   extends:
-    - .zombienet-substrate-common
+    - .zombienet-substrate-warp-sync-common
+  before_script:
+    - !reference [.zombienet-substrate-warp-sync-common, before_script]
+    - cp ${LOCAL_DIR}/0001-basic-warp-sync/chain-spec.json ${LOCAL_DIR}/0002-validators-warp-sync
```

Comment:
hm, running locally requires remembering about this copy. This can become frustrating. Maybe better do a link?